components/input/input.vue

```diff
@@ -556,6 +565,9 @@ export default {
 
   methods: {
     inputClasses () {
+      if (this.hidden) {
+        return [];
+      }
```

Comment:
Why is this necessary, does it break something to set these classes when hidden? the dom elements would still exist right?

Reply:
yea actually i could remove both the return statements.